src/puppetlabs/puppetdb/cli/benchmark.clj

```diff
@@ -434,7 +434,7 @@
     ;; amq send thread
     (future
       (try
-        (let [s (.createSession connection true Session/SESSION_TRANSACTED)
+        (let [s (.createSession connection true 0)
```

Comment:
For reference: http://activemq.apache.org/maven/apidocs/org/apache/activemq/ActiveMQConnection.html#createSession%28boolean,%20int%29